param/_utils.py

```diff
@@ -518,30 +518,54 @@ def _is_number(obj):
     else: return False
 
 
-def _is_abstract(class_):
+def _is_abstract(class_: type) -> bool:
     try:
         return class_.abstract
     except AttributeError:
         return False
```

Comment:
Is this the best current way to determine if a class is abstract? I would have thought it needed multiple tests to handle the (older) Param way of declaring that and the (newer) Python 3 way of declaring that.

Reply:
This is Param's own way of determining if a class is abstract. I added support to the standard way in https://github.com/holoviz/param/pull/1031, I'd be fine updating `_is_abstract` here first/too.

Reply:
Ok I'll cherry-pick that change from https://github.com/holoviz/param/pull/1031.

Reply:
Added in https://github.com/holoviz/param/pull/1027/commits/bfe9c63abcdf774e6aad049455258726c09eb1c0